Dockerfile

```diff
@@ -1,4 +1,15 @@
-FROM golang:1.15-alpine
+FROM node:15.5.1-alpine AS assets-builder
+
+WORKDIR /assets
+
+COPY ./package.json ./package-lock.json ./
+COPY ./conf/webpack/. ./conf/webpack/
+COPY ./assets/. ./assets/
```

Comment:
You should copy tailwind and postCSS configurations as well, otherwise, it could not build the assets properly. Check it on [my PR](https://github.com/hoangmirs/go-scraper/pull/31)